base/array.jl

```diff
@@ -387,8 +387,22 @@ julia> fill(1.0, (5,5))
  1.0  1.0  1.0  1.0  1.0
 ```
 
-If `x` is an object reference, all elements will refer to the same object. `fill(Foo(),
-dims)` will return an array filled with the result of evaluating `Foo()` once.
+If `x` is an object reference, all elements will refer to the same object.
+
+# Examples
+```jldoctest
+julia> X = fill(zeros(2),2)
+2-element Array{Array{Float64,1},1}:
+ [0.0, 0.0]
+ [0.0, 0.0]
+julia> X[1][1] = 99;
```

Comment:
This should have a newline inserted above and below.

Reply:
Sorry for asking (I really don't understand the PR machinery works): do I need to do anything to incorporate your comment?

Reply:
Usually it's possible for reviewer to make a concrete code change suggestion, which can be easily committed, but I didn't manage this time. So yeah, you would need to update the PR but I'm not really sure how, as I see that it's made from an "unknown repository"...

Reply:
@ViralBShah Should I manually combine these changes and make a new PR? Or, is there another way?  /Paul

Reply:
I think making a new PR is the easiest way.